scopesim/effects/spectral_trace_list_utils.py

```diff
@@ -134,7 +141,7 @@ def map_spectra_to_focal_plane(self, fov):
         The method returns a section of the fov image along with info on
         where this image lies in the focal plane.
         """
-
+        print("Mapping", fov.meta['trace_id'])
```

Comment:
```suggestion
        logging.info("Mapping %s" % fov.meta['trace_id'])
```

Reply:
Should be `logging.info("Mapping %s", fov.meta["trace_id"])`, logging functions take `*args` for the formatting.

Reply:
Yeah, nice, except that I don't see the message anymore (admittedly, it's purpose is to keep me awake by providing some progress info). I see that `cmd['!SIM.logging.console_level']` is set to `'WARNING'` (in scopesim's `defaults.yaml`, so I guess that would have to be changed explicitely. Do we have a policy on logging? I usually feel that scopesim is a bit too quiet when doing lengthy operations.
In any case, I adopt it with an associated reformulation of the warning on l.177.